returnn/datasets/postprocessing.py

```diff
@@ -296,7 +298,70 @@ def _make_tensor_template_from_input(self, data_key: str) -> Tensor:
         return Tensor(data_key, dims=dims, dtype=dtype, sparse_dim=sparse_dim)
 
 
-class LaplaceOrdering(Callable[[Iterator[TensorDict]], Iterator[TensorDict]]):
+class SeqOrdering:
```

Comment:
I don't see that this is such generic base class that I would call/use it that way. Maybe better:
```suggestion
class _GetSeqLenByLengthKey:
```